pandas/util/clipboard.py

```diff
@@ -138,7 +138,7 @@ def _copyGtk(text):
 
 
 def _pasteQt():
-    return str(cb.text())
+    return unicode(cb.text())
```

Comment:
This is probably better:
```
    return text_type(cb.text())
```
`text_type` is defined above as `text_type = unicode if PY2 else str`.

Although, I don't know if this has anything to do with other test failures.